examples/nlp/intent_slot_classification/conf/intent_slot_classification_config.yaml

```diff
@@ -0,0 +1,91 @@
+# Sentence classification with pretrained BERT models
+
+pl:
+  trainer:
+    gpus: 1 # the number of gpus, 0 for CPU
+    num_nodes: 1
+    max_epochs: 100
+    max_steps: null # precedence over max_epochs
+    accumulate_grad_batches: 1 # accumulates grads every k batches
+    amp_level: O0 # O1/O2 for mixed precision
+    distributed_backend: ddp
+    checkpoint_callback: False  # Provided by exp_manager
+    logger: False  # Provided by exp_manager
+
+model:
+  data_dir: ??? # /path/to/data
+  class_balancing: null # or weighted_loss
+  intent_loss_weight: 0.6 # only if class_balancing = weighted_loss
+  pad_label: '-1'
+  none_slot_label: 'O'
+  ignore_extra_tokens: false
+  ignore_start_end: false
+
+  train_ds:
+    prefix: train
+    batch_size: 32
+    shuffle: true
+    num_samples: -1
+    num_workers: 2
+    drop_last: false
+    pin_memory: false
+
+  validation_ds:
+    prefix: dev
+    batch_size: 32
+    shuffle: false
+    num_samples: -1
+    num_workers: 2
+    drop_last: false
+    pin_memory: false
+
+  language_model:
+    pretrained_model_name: bert-base-uncased
+    max_seq_length: 50
+    bert_checkpoint: null
+    bert_config: null
+    tokenizer: nemobert # 'nemobert' or 'sentencepiece'
+    tokenizer_model: null # only used if tokenizer is sentencepiece
+    do_lower_case: true # true for uncased models, false for cased models, will be set automatically if pre-trained tokenizer model is used
+  
+  head:
+    num_output_layers: 2
+    fc_dropout: 0.1
+
+  optim:
```

Comment:
Please update the param format to the new one. I have a new PR with the new format.

Reply:
Let me do this step afterwards. I want to push working code to Candidates.